### PR TITLE
feat(chip): disabled state for chip

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
## **Describe pull-request**  
Adding disabled state for chip aligning with already existing design. 
## **Issue Linking:**  
- **Jira:** [CDEP-3238](https://tegel.atlassian.net/browse/CDEP-3238)    

## **How to test**  
1. Go to Storybook
2. Check in Chip
3. Toggle disabled boolean and see that it applies correctly for all types and aligns with Figma design.

[CDEP-3238]: https://tegel.atlassian.net/browse/CDEP-3238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/components/chip/chip.tsx [hunk 1]
```diff
@@ -38,6 +38,9 @@ export class TdsChip {
   /** Value of input. Valid only for type checkbox and radio. */
   @Prop() value: string;
 
+  /** Sets the Chiop in a disabled state */
```
Comment (timrombergjakobsson) [inline packages/core/src/components/chip/chip.tsx:41]:
```suggestion
  /** Sets the Chip in a disabled state */
```

File: packages/core/src/components/chip/chip-vars.scss [hunk 2]
```diff
@@ -15,6 +18,9 @@
   --tds-chips-color: var(--tds-white);
   --tds-chips-color-active: var(--tds-white);
   --tds-chips-background: var(--tds-grey-800);
+  --tds-chips-background-disabled: var(--tds-grey-200);
```
Comment (theJohnnyMe) [inline packages/core/src/components/chip/chip-vars.scss:21]:
```suggestion
  --tds-chips-background-disabled: var(--tds-grey-800);
```
Seems like 800 is one for dark mode. Please double check.

Comment (timrombergjakobsson) [reply packages/core/src/components/chip/chip-vars.scss:21]: Good catch!

**Review by theJohnnyMe — APPROVED** [review]
Nice work Tim! 
LGTM

File: packages/core/src/components/chip/chip.tsx [hunk 3]
```diff
@@ -87,21 +91,30 @@ export class TdsChip {
...
+      // Only proceed if not disabled
+      this.tdsClick.emit({
+        chipId: this.chipId,
+      });
+    }
   };
 
   private renderInputAttributes() {
+    const commonAttributes = {
+      disabled: this.disabled, // Adding disabled attribute
```
Comment (timrombergjakobsson) [inline packages/core/src/components/chip/chip.tsx:104]:
```suggestion
      disabled: this.disabled
```

**Review by mistermalm — APPROVED** [review]

**Pull request merged**